mathjax3-ts/output/chtml/Wrappers/scriptbase.ts

```diff
@@ -79,7 +84,10 @@ export class CHTMLscriptbase<N, T, D> extends CHTMLWrapper<N, T, D> {
         //
         let core = this.baseCore = this.childNodes[0];
         if (!core) return;
-        while (core.childNodes.length === 1 && (core.node.isKind('mrow') || core.node.isKind('TeXAtom'))) {
+        while (core.childNodes.length === 1 &&
+               (core.node.isKind('mrow') || core.node.isKind('TeXAtom') ||
+                core.node.isKind('mstyle') || core.node.isKind('mpadded') ||
+                core.node.isKind('mphantom') || core.node.isKind('semantics'))) {
```

Comment:
Shorten by testing if kind is element of the list [mrow, TeXAtom....

Reply:
The `isKind()` test is actually performing `instanceof`, not a string test, so these can't be combined in that way.  This way, they will handle subclasses, since that is our new mechanism of handling extensions.  

I suppose there could be a `kindIsOneOf(...names: string[])` to test multiple kinds at once, if you would prefer that.

Reply:
If that means we still have to do the function calls, than it is not worth it. So leave as is.